packages/components/psammead-story-promo/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 8.0.32 | [PR#4588](https://github.com/bbc/psammead/pull/4588) Fix TalkBack reading comma bug |
```

Comment:
This fixes 'TalkBack incorrectly reading links in nested spans' not the comma bug?

Reply:
Yes, this is my mistake, thank you for spotting this :)